app/Routes.js

```diff
@@ -111,26 +107,59 @@ export const Routes = (
   </div>
 );
 
-const WalletsSubpages = (stores, actions) => (
-  <Switch>
-    <Route
-      exact
-      path={ROUTES.WALLETS.TRANSACTIONS}
-      component={(props) => <WalletSummaryPage {...props} stores={stores} actions={actions} />}
-    />
-    <Route
-      exact
-      path={ROUTES.WALLETS.SEND}
-      component={(props) => <WalletSendPage {...props} stores={stores} actions={actions} />}
-    />
-    <Route
-      exact
-      path={ROUTES.WALLETS.RECEIVE}
-      component={(props) => <WalletReceivePage {...props} stores={stores} actions={actions} />}
-    />
-    <Redirect to={ROUTES.WALLETS.TRANSACTIONS} />
-  </Switch>
-);
+const WalletsSubpages = (stores, actions) => {
+  let redirectRoute = undefined;
+  let routes = undefined;
+
+  if (environment.isShelley()) {
+    redirectRoute = ROUTES.WALLETS.STAKE_DASHBOARD;
+    routes = (
+      <>
+        <Route
+          exact
+          path={ROUTES.WALLETS.STAKE_DASHBOARD}
+          component={(props) => <StakingDashboardPage {...props} stores={stores} actions={actions} />}
+        />
+        <Route
+          exact
+          path={ROUTES.WALLETS.STAKE_SIMULATOR}
+          component={(props) => <StakingSimulatorPage {...props} stores={stores} actions={actions} />}
+        />
+        <Route
+          exact
+          path={ROUTES.WALLETS.STAKE_ADVANCED_SIMULATOR}
+          component={(props) => <StakingAdvancedSimulatorPage {...props} stores={stores} actions={actions} />}
+        />
+      </>);
+  } else {
+    redirectRoute = ROUTES.WALLETS.TRANSACTIONS;
```

Comment:
PR is fine but I don't think we need to remove these anymore now that the key issues are all resolved and we have more time to ship support for this @nicarq

Reply:
@SebastienGllmt 
Re-enabled.